PWGHF/TableProducer/HFTreeCreatorLbToLcPi.cxx

```diff
@@ -0,0 +1,219 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file HFTreeCreatorLbToLcPi.cxx
+/// \brief Writer of the 2 prong candidates in the form of flat tables to be stored in TTrees.
+///        Intended for debug or for the local optimization of analysis on small samples.
+///        In this file are defined and filled the output tables
+/// \note Extended from HFTreeCreatorD0ToKPi, HFTreeCreatorLcToPKPi, HFTreeCreatorXToJpsiPiPi
+///
+/// \author Panos Christakoglou <Panos.Christakoglou@cern.ch>, Nikhef
```

Comment:
Is this correct?

Reply:
Most of this working version was written by Panos, I tweeked some minor things. If appropriate, I can add myself as well

Reply:
I leave it up to you then. In case you decide to add yourself, please do so by adding a separate `/// \author` line.